app/models/early_career_payments/slug_sequence.rb

```diff
@@ -101,6 +103,8 @@ def slugs
 
         sequence.delete("teacher-reference-number") if claim.logged_in_with_tid? && claim.teacher_reference_number.present?
 
+        remove_qualification_questions_slugs(sequence) if claim.logged_in_with_tid? && !session[:page_sequence_flag]
```

Comment:
Hi @slorek,
The second check is there to see if the data is coming from the DQT API. If we just remove these three pages based on the 'logged_in_with_tid' flag, it will cause an issue at this line of code:
```
if params[:slug] == "teaching-subject-now" && !current_claim.eligibility.eligible_itt_subject 
    return redirect_to claim_path(current_policy_routing_name, "eligible-itt-subject") 
```
This problem happens because the value of 'current_claim.eligibility.eligible_itt_subject' will be empty, and the redirect path won't exist because we removed the 'eligible-itt-subject' slug from the sequence.